### PR TITLE
Release 2.5.1

### PR DESCRIPTION
**Pull request by yorickdewid**
Release version 2.5.1

### CODE REVIEW

File: src/FunderMaps.Portal/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "ConnectionStrings": {
+    "FunderMapsConnection": null
+  },
+  "BlobStorage": {
+    "AccessKey": null,
+    "SecretKey": null,
+    "ServiceUri": null,
+    "BlobStorageName": null
+  }
+}
```
Comment (tabeckers) [inline src/FunderMaps.Portal/appsettings.json:31]: Line end

```suggestion
}

```

File: src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs [hunk 2]
```diff
@@ -22,184 +20,264 @@ namespace FunderMaps.WebApi.Controllers.Report
...
-            await _inquiryUseCase.UpdateStatusAsync(id, AuditStatus.PendingReview, input.Message);
+            var inquiry = await _inquiryRepository.GetByIdAsync(id);
+
+            // Transition.
+            inquiry.State.TransitionToReview();
+
+            // Act.
+            await _inquiryRepository.SetAuditStatusAsync(inquiry.Id, inquiry);
+            await _notificationService.NotifyByEmailAsync(
+                address: new string[] { "info@example.com" }, // TODO:
```
Comment (tabeckers) [inline src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs:220]: Don't we want this non-hardcoded for a master merge?

**Review by tabeckers — CHANGES_REQUESTED** [review]
See comments

File: src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs (hunk 2 above)
Comment (yorickdewid) [reply src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs:220]: True, and we do have a configuration option for this value. However the structure is not yet final

**Review by tabeckers — APPROVED** [review]

File: src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs (hunk 2 above)
Comment (tabeckers) [reply src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs:220]: Check

**Pull request merged**